README.md

```diff
@@ -65,52 +90,85 @@ inspect or change it and save it out to file. Check out the
 [example jupyter notebooks](https://github.com/arup-group/genet/tree/master/notebooks) 
 for usage examples.
 
-#### Install dev prereqs (use equivalent linux or windows package management)
+
+### Installation as a Python Package
+**Note:** if you plan only to _use_ GeNet rather than make code changes to it, you can avoid having to perform any
+local installation by using [GeNet's Docker image](#using-docker). If you are going to make code changes, or you don't
+want to use Docker for some reason...
+
+#### Native dependencies
+GeNet uses some Python libraries that rely on underlying native libraries for things like geospatial calculations and
+linear programming solvers. Before you install GeNet's Python dependencies, you must first install these native
+libraries.
+
+
+#### A note on the mathematical solver
+**Note**: The default CBC solver is pre-installed inside [GeNet's Docker image](#using-docker), which can save you some
+installation effort
+
+To use methods which snap public transit to the graph, GeNet uses a mathematical solver. If you won't be using such
+functionality, you do not need to install this solver.
+Methods default to [CBC](https://projects.coin-or.org/Cbc), an open source solver.
+Another good open source choice is [GLPK](https://www.gnu.org/software/glpk/).
+The solver you use needs to support MILP - mixed integer linear programming.
+
+#### Installing the native dependencies
+The commands for installing the necessary native libraries vary according to the operating system you are using, for
+example:
+
+| OS       | Commands |
```

Comment:
I dunno what it is Windows users do...

Reply:
yeah, I don't know either, i think a lot of people use a linux subsystem